src/entities/applet/model/mappers.ts

```diff
@@ -89,7 +89,7 @@ export function mapAppletDetailsFromDto(
     encryption: detailsDto.encryption,
     streamEnabled: detailsDto.streamEnabled,
     streamIpAddress: detailsDto.streamIpAddress,
-    streamPort: detailsDto.streamPort,
+    streamPort: detailsDto.streamPort ? `${detailsDto.streamPort}` : null,
```

Comment:
If the intent is to cast a number value to string then please consider this piece of code code.
```suggestion
    streamPort: detailsDto?.streamPort?.toString() ??  null,
```